setup.py

```diff
@@ -19,6 +19,7 @@ def get_version() -> str:
     "typing-extensions>=3.7.4.3",  # to be able to import TypeAlias
     "importlib_metadata;python_version<'3.8'",
     "packaging>=20.9",
+    "Jinja2",
```

Comment:
Opted to just put it in `testing` requirements. Since Jinja is only required by `from_template` and its only one dep, I figure its a bit overkill to add a whole new extras category.

Reply:
Should this be removed from `install_requires` ?

Reply:
Whoops! you're right.